### PR TITLE
added to implementation and unit tests

### PR DESCRIPTION
**Pull request by steven-xu-lf**
issue -> #33

### CODE REVIEW

File: src/main/java/com/laserfiche/api/client/model/AccessKey.java [hunk 1]
```diff
@@ -27,6 +27,10 @@ public static AccessKey CreateFromBase64EncodedAccessKey(String base64EncodedAcc
 
             mapper.registerModule(module);
         }
+        base64EncodedAccessKey = base64EncodedAccessKey.trim();
+        if (base64EncodedAccessKey.length() == 0) {
+            throw new RuntimeException("Input cannot be empty or null");
```
Comment (lf-jason-jia) [inline src/main/java/com/laserfiche/api/client/model/AccessKey.java]: I think we could have some more sepcific exception types, like illegal argument exception (or something similar).

File: src/test/java/com/laserfiche/api/client/unit/AccessKeyTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Objects;
+
+class AccessKeyTest {
+
+    @Test
+    void CreateFromBase64EncodedAccessKey_ValidBase64() throws Exception {
+        //Arrange
```
Comment (lf-jason-jia) [inline src/test/java/com/laserfiche/api/client/unit/AccessKeyTest.java]: While not strictly required, but would be nice if there a space between `//` and the text. See: https://google.github.io/styleguide/javaguide.html

File: src/main/java/com/laserfiche/api/client/model/AccessKey.java (hunk 1 above)
Comment (steven-xu-lf) [reply src/main/java/com/laserfiche/api/client/model/AccessKey.java]: ok changed it to illegal argument exception

File: src/test/java/com/laserfiche/api/client/unit/AccessKeyTest.java (hunk 2 above)
Comment (steven-xu-lf) [reply src/test/java/com/laserfiche/api/client/unit/AccessKeyTest.java]: ok done along with fixing some code smells

**Review by lf-jason-jia — APPROVED** [review]
It passed Boomi's environment!

**Pull request merged**